src/components/SnazzyButton.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { CSSProperties } from 'react';
+import { Button, ButtonProps } from '@patternfly/react-core';
+import { ColoredTd } from './ColoredTd';
+
+type SnazzyButtonProps = {
+  color: string;
+  isSnazzy: boolean;
+  customStyle?: string;
+} & ButtonProps;
+
+const SnazzyButton: React.FC<SnazzyButtonProps> = ({ color, isSnazzy, customStyle, ...rest }) => {
+  if (isSnazzy) {
+    return (
+      <ColoredTd color={color}>
+        <Button style={customStyle as CSSProperties} {...rest} />
```

Comment:
This is good, but reread the task, we want to use classname/react-jss/useStyles to add our style.